xapian-maintainer-tools/profiling/profiler.c

```diff
@@ -8,9 +8,9 @@
  * xapian-io-profile script, then use:
  * ./xapian-io-profile --log=log_file_name ./executable
  *
- * Running this library will produce an strace.log file,
+ * Running this library will produce a log file,
  * which can be fed to strace-analyse to produce a log, by running:
- * ./strace-analyse strace.log
+ * ./strace-analyse log_file_name
```

Comment:
You shouldn't make two different changes in one commit - the comment updates are useful, but aren't part of implementing the caching of function pointers.  For more advice on structuring commits, see:

https://xapian-developer-guide.readthedocs.io/en/latest/contributing/workflow.html#good-commits

Reply:
Force pushed into 2 separate commits.